README.md

```diff
@@ -327,6 +327,9 @@ Use this repo to share and keep track of software, tech, CS, PM, quant internshi
 |[Intuitive](https://careers.intuitive.com/us/en/job/214284/Computer-Science-Computer-Engineering-Intern) | Sunnyvale, CA | Computer Science/Computer Engineering Intern. |
 |[Twitter](https://jobs.smartrecruiters.com/ni/Twitter2/f6205ccb-ee15-4669-b8f4-0b773c974c02-2022-engineering-internships-north-america-latin-america) | San Francisco, CA | SWE, Machine Learning, Data Science internships and more! |
 |[Workday](https://workday.wd5.myworkdayjobs.com/en-US/Workday_University_Careers) | Various | SDE, SDET, Application, BA, Product, etc. |
+|[Epic Games](https://www.epicgames.com/site/en-US/students) | US | Various intern positions in software engineering, game design, analytics, etc.|
+|[Rivian](https://rivian.com/careers?department=internships) | US, Canada | Software Engineering Intern |
```

Comment:
I think there exists a rivian entry alr

Reply:
Oh sorry I missed that! I deleted it from the PR!